File: computer_science/99problems/go/1.01.go
package main

func ttyFindLast(list []string) {
	length := len(list)
	last := list[length-1]

	if length == 0 {
		println("The list is empty.")
	} else {
		println(last)
	}
}

func main() {
	list := []string{"a", "b", "c", "d"}

	ttyFindLast(list)
}
// => d

File: computer_science/99problems/c/1.01.c
#include <stdio.h>

void tty_find_last(char list[], size_t length)
{
	char last = list[length - 1];

	if (last == 0)
		printf("The list is empty.\n");
	else
		printf("%c\n", last);
}

int main()
{
	char list[] = {'a', 'b', 'c', 'd'};
	size_t length = sizeof(list) / sizeof(list[0]);

	tty_find_last(list, length);
	return 0;
}
/* => d  */
